### PR TITLE
1st change edge ver from i64 to char; 

### PR DESCRIPTION
**Pull request by liuyu85cn**
change format of vertex & edge key

1st change edge ver from i64 to char;

2nd erase vertex ver

### CODE REVIEW

File: src/storage/mutate/DeleteEdgesProcessor.cpp [hunk 1]
```diff
@@ -67,7 +67,7 @@ void DeleteEdgesProcessor::process(const cpp2::DeleteEdgesRequest& req) {
                                                    edgeKey.edge_type,
                                                    edgeKey.ranking,
                                                    edgeKey.dst.getStr(),
-                                                   std::numeric_limits<int64_t>::max());
+                                                   2);
```
Comment (critical27) [inline src/storage/mutate/DeleteEdgesProcessor.cpp:70]: better use numeric_limits

File: src/storage/transaction/TransactionManager.cpp [hunk 2]
```diff
@@ -260,7 +261,15 @@ folly::Future<cpp2::ErrorCode> TransactionManager::resumeTransaction(size_t vIdL
     // 2nd, get values from remote in-edge
     auto spPromiseVal = std::make_shared<cpp2::ErrorCode>(cpp2::ErrorCode::SUCCEEDED);
     auto c = folly::makePromiseContract<cpp2::ErrorCode>();
-    auto remoteKey = reverseKey(vIdLen, spaceId, localKey, ver);
+
+    // auto remoteKey = reverseKey(vIdLen, spaceId, localKey, ver);
```
Comment (critical27) [inline src/storage/transaction/TransactionManager.cpp:265]: could remove it?

**Review by critical27 — COMMENTED** [review]
Generally LGTM

**Review by critical27 — APPROVED** [review]
LGTM

**Comment by critical27** [comment]
One extra place need to handle, `keyWithNoVersion`

File: src/tools/db-upgrade/NebulaKeyUtilsV1.h [hunk 3]
```diff
@@ -226,7 +225,7 @@ class NebulaKeyUtilsV1 final {
 
     static constexpr int32_t kEdgeLen = sizeof(PartitionID) + sizeof(VertexID)
                                       + sizeof(EdgeType) + sizeof(VertexID)
-                                      + sizeof(EdgeRanking) + sizeof(EdgeVersion);
+                                      + sizeof(EdgeRanking) + sizeof(EdgeVerPlaceHolder);
```
Comment (panda-sheep) [inline src/tools/db-upgrade/NebulaKeyUtilsV1.h:228]: This is v1, it should not be changed？

Comment (liuyu85cn) [reply src/tools/db-upgrade/NebulaKeyUtilsV1.h]: good catch

**Comment by liuyu85cn** [comment]
build (centos7, gcc-9.2) Expected — Waiting for status to be reported

**Review by critical27 — APPROVED** [review]
Good job, LGTM

**Review by darionyaphet — APPROVED** [review]
novus mundus

**Pull request merged**